src/common/lib/api/index.js

```diff
@@ -33,6 +33,11 @@ apiClientRequest.interceptors.response.use(
   error => {
     Sentry.captureException(error);
 
+    if (error?.response?.status === 503 || !error?.response) {
+      VueRouter.push({ name: "maintenance" })
```

Comment:
Is it possible to make this a popup instead of a separate page?

You can use VueStore to store the page error values and to show a pop-up, and access the VueStore like this: https://github.com/debionetwork/debio-customer-ui/blob/main/src/common/lib/route-guard.js#L19

The popup can be housed in the Main page layout like somewhere in here: https://github.com/debionetwork/debio-customer-ui/blob/main/src/views/LandingPage/Layout.vue

Reply:
Good idea. Of course, I can implement this using your idea, but I think will be good to create a new card in the next sprint. Wdyt?

Reply:
That's a great idea, no work should go unwritten @bukandicki 

@hilyds please note.